### PR TITLE
SciPy 2019

### PR DESCRIPTION
**Pull request by akielbowicz**
```yml
- title: SciPy 2019
  dir: scipy-2019
  youtube_list:
    - https://www.youtube.com/playlist?list=PLYx7XA2nY5GcDQblpQ_M1V3PQPoLWiDAC
  related_urls:  
    - label: Conference schedule
      url: https://www.scipy2019.scipy.org/confschedule
  language: eng
  dates:
    begin: 2019-07-09
    end: 2019-07-12
    default: 2019-07-09
  issue: 
  minimal_download: True
  tags: []
```

Also the speakers, tags and titles are completed.

### CODE REVIEW

File: scipy-2019/videos/rapids-open-gpu-data-science-scipy-2019-tutorial-scopatz-becker-kraus-gama-dessavre.json [hunk 1]
```diff
@@ -10,10 +10,17 @@
       "url": "https://www.scipy2019.scipy.org/confschedule"
     }
   ],
-  "speakers": [],
-  "tags": [],
+  "speakers": [
+    "Scopatz",
```
Comment (di) [inline scipy-2019/videos/rapids-open-gpu-data-science-scipy-2019-tutorial-scopatz-becker-kraus-gama-dessavre.json]:
```suggestion
    "Anthony Scopatz",
```

File: scipy-2019/videos/rapids-open-gpu-data-science-scipy-2019-tutorial-scopatz-becker-kraus-gama-dessavre.json [hunk 2]
```diff
@@ -10,10 +10,17 @@
...
+    "Becker",
```
Comment (di) [inline scipy-2019/videos/rapids-open-gpu-data-science-scipy-2019-tutorial-scopatz-becker-kraus-gama-dessavre.json]:
```suggestion
    "Nick Becker",
```

File: scipy-2019/videos/rapids-open-gpu-data-science-scipy-2019-tutorial-scopatz-becker-kraus-gama-dessavre.json [hunk 3]
```diff
@@ -10,10 +10,17 @@
...
+    "Kraus",
```
Comment (di) [inline scipy-2019/videos/rapids-open-gpu-data-science-scipy-2019-tutorial-scopatz-becker-kraus-gama-dessavre.json]:
```suggestion
    "Keith Kraus",
```

File: scipy-2019/videos/rapids-open-gpu-data-science-scipy-2019-tutorial-scopatz-becker-kraus-gama-dessavre.json [hunk 4]
```diff
@@ -10,10 +10,17 @@
...
+    "Gama Dessavre"
```
Comment (di) [inline scipy-2019/videos/rapids-open-gpu-data-science-scipy-2019-tutorial-scopatz-becker-kraus-gama-dessavre.json]:
```suggestion
    "Dante Gama Dessavre"
```

File: scipy-2019/videos/scipy-tools-plenary-session-scipy-2019.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  "language": "eng",
+  "recorded": "2019-07-11",
+  "related_urls": [
+    {
+      "label": "Conference schedule",
+      "url": "https://www.scipy2019.scipy.org/confschedule"
+    }
+  ],
+  "speakers": [
+    "Various Speakers"
```
Comment (di) [inline scipy-2019/videos/scipy-tools-plenary-session-scipy-2019.json]: Just leave this as an empty list.

File: scipy-2019/videos/the-jupyter-interactive-widget-ecosystem-scipy-2019-tutorial-breddels-craig-grout-renou.json [hunk 6]
```diff
@@ -10,10 +10,17 @@
       "url": "https://www.scipy2019.scipy.org/confschedule"
     }
   ],
-  "speakers": [],
-  "tags": [],
+  "speakers": [
+    "Breddels",
```
Comment (di) [inline scipy-2019/videos/the-jupyter-interactive-widget-ecosystem-scipy-2019-tutorial-breddels-craig-grout-renou.json]:
```suggestion
    "Maarten Breddels",
```

File: scipy-2019/videos/the-jupyter-interactive-widget-ecosystem-scipy-2019-tutorial-breddels-craig-grout-renou.json [hunk 7]
```diff
@@ -10,10 +10,17 @@
...
+    "Craig",
```
Comment (di) [inline scipy-2019/videos/the-jupyter-interactive-widget-ecosystem-scipy-2019-tutorial-breddels-craig-grout-renou.json]:
```suggestion
    "Matt Craig",
```

File: scipy-2019/videos/the-jupyter-interactive-widget-ecosystem-scipy-2019-tutorial-breddels-craig-grout-renou.json [hunk 8]
```diff
@@ -10,10 +10,17 @@
...
+    "Grout",
```
Comment (di) [inline scipy-2019/videos/the-jupyter-interactive-widget-ecosystem-scipy-2019-tutorial-breddels-craig-grout-renou.json]:
```suggestion
    "Jason Grout",
```

File: scipy-2019/videos/the-jupyter-interactive-widget-ecosystem-scipy-2019-tutorial-breddels-craig-grout-renou.json [hunk 9]
```diff
@@ -10,10 +10,17 @@
...
+    "Renou"
```
Comment (di) [inline scipy-2019/videos/the-jupyter-interactive-widget-ecosystem-scipy-2019-tutorial-breddels-craig-grout-renou.json]:
```suggestion
    "Martin Renou"
```

File: scipy-2019/videos/challenges-in-detecting-physiological-changes-using-wearable-sensor-data-scipy-2019-e-preble.json [hunk 10]
```diff
@@ -10,10 +10,18 @@
...
     }
   ],
-  "speakers": [],
+  "speakers": [
+    "Maria Davila-Hernandez",
+    "Kristin Gilchrist",
+    "Meghan Hegarty-Craver",
+    "Michele Hernandez",
+    "Gregory Lewis",
+    "Edward Preble",
```
Comment (Daniel-at-github) [inline scipy-2019/videos/challenges-in-detecting-physiological-changes-using-wearable-sensor-data-scipy-2019-e-preble.json]: Seems that Edward Preble is the speaker and the rest are collaborators not seen in the video.

File: scipy-2019/videos/dashboarding-with-jupyter-notebooks-voila-and-widgets-scipy-2019.json [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+    "Maarten Breddels",
+    "Martin Renou"
+  ],
+  "tags": [],
+  "thumbnail_url": "https://i.ytimg.com/vi/VtchVpoSdoQ/hqdefault.jpg",
+  "title": "Dashboarding with Jupyter Notebooks, Voila and Widgets",
+  "videos": [
+    {
+      "type": "youtube",
+      "url": "https://www.youtube.com/watch?v=qyJXBlrdzBs"
```
Comment (Daniel-at-github) [inline scipy-2019/videos/dashboarding-with-jupyter-notebooks-voila-and-widgets-scipy-2019.json]:
```suggestion
      "url": "https://www.youtube.com/watch?v=VtchVpoSdoQ"
```

File: scipy-2019/videos/day-1-lightning-talks-scipy-2019.json [hunk 12]
```diff
@@ -10,10 +10,12 @@
       "url": "https://www.scipy2019.scipy.org/confschedule"
     }
   ],
-  "speakers": [],
+  "speakers": [
+    "Various Speakers"
```
[the remark this answers is not in the document]
Comment (Daniel-at-github) [reply scipy-2019/videos/day-1-lightning-talks-scipy-2019.json]: In lightning talks speakers: [ "Various Speakers" ] is the usual representation.
Why? ... Too much work (sometimes done with tools/csv2rst.py help), and not always you can extract all the names from the video.
I suppose that is a way to mark it as "we worked on it" and to polish it further you need to walk an extra mile.

If this don't make sense, it's easy to convert: lightning talks + Various Speakers <=> lightning talks + Unknown

File: scipy-2019/videos/skorch-a-union-of-scikit-learn-and-pytorch-scipy-2019-thomas-fan.json [hunk 13]
```diff
@@ -10,7 +10,9 @@
       "url": "https://www.scipy2019.scipy.org/confschedule"
     }
   ],
-  "speakers": [],
+  "speakers": [
+    "Thomas Fan"
+  ],
   "tags": [],
   "thumbnail_url": "https://i.ytimg.com/vi/0J7FaLk0bmQ/maxresdefault.jpg",
   "title": "Skorch: A Union of Scikit learn and PyTorch | SciPy 2019 | Thomas Fan",
```
Comment (Daniel-at-github) [inline scipy-2019/videos/skorch-a-union-of-scikit-learn-and-pytorch-scipy-2019-thomas-fan.json]:
```suggestion
   "title": "Skorch: A Union of Scikit learn and PyTorch",
```

File: scipy-2019/videos/to-a-billion-beyond-how-to-visually-explore-compare-share-large-quant-datasets-w-higlass.json [hunk 14]
```diff
@@ -10,7 +10,11 @@
...
   ],
-  "speakers": [],
+  "speakers": [
+    "Nezar Abdennur",
+    "Peter Kerpedjiev",
+    "Fritz Lekschas"
+  ],
   "tags": [],
   "thumbnail_url": "https://i.ytimg.com/vi/v62k4Ok1S8g/maxresdefault.jpg",
   "title": "To a Billion & Beyond: How to Visually Explore, Compare & Share Large Quant. Datasets w/ HiGlass",
```
Comment (Daniel-at-github) [inline scipy-2019/videos/to-a-billion-beyond-how-to-visually-explore-compare-share-large-quant-datasets-w-higlass.json]:
```suggestion
   "title": "To a Billion and Beyond: How to Visually Explore, Compare and Share Large Quantitative Datasets with HiGlass",
```

File: scipy-2019/videos/xgcm-analyzing-general-circulation-models-in-python-scipy-2019-ryan-abernathey.json [hunk 15]
```diff
@@ -10,10 +10,13 @@
...
   ],
-  "speakers": [],
+  "speakers": [
+    "Ryan Abernathey",
+    "Julius Busecke"
+  ],
   "tags": [],
   "thumbnail_url": "https://i.ytimg.com/vi/lo6JvrJ-YA8/maxresdefault.jpg",
-  "title": "Xgcm: Analyzing General Circulation Models in Python | SciPy 2019 | Ryan Abernathey",
+  "title": "Xgcm: Analyzing General Circulation Models in Pythonc",
```
Comment (Daniel-at-github) [inline scipy-2019/videos/xgcm-analyzing-general-circulation-models-in-python-scipy-2019-ryan-abernathey.json]:
```suggestion
  "title": "Xgcm: Analyzing General Circulation Models in Python",
```

**Comment by Daniel-at-github** [comment]
@akielbowicz, could you batch commit all suggestions? I tried but I have no permissions.

File: scipy-2019/videos/day-1-scipy-tools-plenary-session-scipy-2019.json [hunk 16]
```diff
@@ -10,10 +10,12 @@
       "url": "https://www.scipy2019.scipy.org/confschedule"
     }
   ],
-  "speakers": [],
+  "speakers": [
+    "Various Speakers"
```
[the remark this answers is not in the document]
Comment (akielbowicz) [reply scipy-2019/videos/day-1-scipy-tools-plenary-session-scipy-2019.json]: Ok, thanks for the review and suggestions

File: scipy-2019/videos/challenges-in-detecting-physiological-changes-using-wearable-sensor-data-scipy-2019-e-preble.json (hunk 10 above)
Comment (akielbowicz) [reply scipy-2019/videos/challenges-in-detecting-physiological-changes-using-wearable-sensor-data-scipy-2019-e-preble.json]: Ok, now its only Edward Preble. Thanks for noticing that.

**Comment by akielbowicz** [comment]
@Daniel-at-github done, now can be merged. 
Thanks for the review.

**Comment by Daniel-at-github** [comment]
Thanks for your contribution @akielbowicz (and sorry for the low availability).

**Pull request merged**